python/scannerpy/sink.py

```diff
@@ -93,26 +93,42 @@ def __init__(self, db):
         self._db = db
 
     def __getattr__(self, name):
-        # This will raise an exception if the source does not exist.
-        sink_info = self._db._get_sink_info(name)
-
-        def make_sink(*args, **kwargs):
-            inputs = []
-            if sink_info.variadic_inputs:
-                inputs.extend(args)
-            else:
-                for c in sink_info.input_columns:
-                    val = kwargs.pop(c.name, None)
-                    if val is None:
-                        raise ScannerException(
-                            'sink {} required column {} as input'
-                            .format(name, c.name))
-                    inputs.append(val)
-
-            sink_args = kwargs.pop('args', kwargs)
-            sink = Sink(self._db, name,
-                        inputs,
-                        kwargs if args is None else sink_args)
-            return sink
-
-        return make_sink
+        # Use Sequence as alias of Column
+        if name == 'Sequence' or name == 'FrameSequence':
+            name = name.replace('Sequence', 'Column')
+            def make_sink(*args, **kwargs):
+                column_name = 'frame' if 'Frame' in name else 'column'
+                return Sink(self._db, name, [], dict(columns={column_name: args[0]}))
+            return make_sink
+        else:
+            # This will raise an exception if the source does not exist.
+            sink_info = self._db._get_sink_info(name)
+
+            def make_sink(*args, **kwargs):
+                inputs = []
+                if sink_info.variadic_inputs:
+                    inputs.extend(args)
+                else:
+                    for c in sink_info.input_columns:
+                        val = kwargs.pop(c.name, None)
+                        if val is None:
+                            raise ScannerException(
+                                'sink {} required column {} as input'
+                                .format(name, c.name))
```

Comment:
It's a little annoying that this error will pop up when we aren't exposing sinks/sources explicitly in the current interface. Is there a better way to show this error? Or should it never come up if not using sinks/sources explicitly?

Reply:
It should only come up if the `Storage` classes are implemented incorrectly.